src/Nethermind/Nethermind.Evm/Tracing/GethStyle/Custom/JavaScript/GethLikeBlockJavaScriptTracer.cs

```diff
@@ -33,7 +33,7 @@ public GethLikeBlockJavaScriptTracer(IWorldState worldState, IReleaseSpec spec,
 
     public override void StartNewBlockTrace(Block block)
     {
-        _engines = new ArrayPoolList<IDisposable>(block.Transactions.Length + 1);
+        _engines = new List<IDisposable>(block.Transactions.Length + 1);
```

Comment:
What was the problem with using ArrayPoolList as long as we dispose it? I guess we will have worse performance if we switch to a normal list

Reply:
It was hard to properly dispose it in this case.